server/services/netatmo/lib/utils/device.js

```diff
@@ -0,0 +1,98 @@
+const get = require('get-value');
+
+/**
+ * @description Get Device param by name.
+ * @param {Object} device - Device Object to parse.
+ * @param {string} paramName - The name of the param to get.
+ * @returns {string} Return param.
+ * @example
+ * const value = getDeviceParam({
+ *  params: [{ name: 'test', value: 1 }]
+ * }, 'test');
+ */
+function getDeviceParam(device, paramName) {
+  if (!get(device, 'params')) {
+    return null;
+  }
+  const param = device.params.find((oneParam) => oneParam.name === paramName);
+  if (param) {
+    return param.value;
+  }
+  return null;
+}
+
+/**
+ * @description Set Device param by name.
+ * @param {Object} device - Device Object to parse.
+ * @param {string} paramName - The name of the param to get.
+ * @param {string} newValue - The value to set.
+ * @example
+ * setDeviceParam({
+ *  params: [{ name: 'test', value: 1 }]
+ * }, 'test', 'new-value');
+ */
+function setDeviceParam(device, paramName, newValue) {
```

Comment:
We already have a setDeviceParam and setDeviceFeature in a common utils file. Can you use this? `/server/utils/setDeviceParam.js`
And add your new method.
It can be used be many other services, all do this.

Reply:
Hi !! 
Thank you @atrovato for your review.  Well indeed all these points are not normal, it was supposed to be modified and corrected on an old merge.  I think there was a problem between our different merges. The files are fine with me and have no errors.  Sorry for the time wasted.  I correct and merge / push again tomorrow.

Reply:
Ok sorry, files deleted on an other push here : https://github.com/GladysAssistant/Gladys/pull/1015/commits/04c43e2e14bf195b0a898bbeb77610e621bf10a2